tests/FSharp.Compiler.ComponentTests/FSharpChecker/TransparentCompiler.fs

```diff
@@ -800,4 +800,41 @@ module Stuff =
 
         //Assert.Equal<string>(hash, hash2)
 
-        ()
\ No newline at end of file
+        ()
+
+/// Update these paths to a local response file with compiler arguments of existing F# projects.
+/// References projects are expected to have been built.
+let localResponseFiles =
+    [|
+        @"C:\Projects\fantomas\src\Fantomas.Core.Tests\Fantomas.Core.Tests.rsp"
```

Comment:
Nothing against Fantomas and not necessarily for now - I prefer having such hardcoded paths at the highest level possible, at least at the test level. Otherwise those things tend to go deep in the codebase and cause confusion - I was just dealing with this recently when reviving benchmarks.

Reply:
Please elaborate on how you would like it.

Reply:
> Nothing against Fantomas and not necessarily for now - I prefer having such hardcoded paths at the highest level possible, at least at the test level. Otherwise those things tend to go deep in the codebase and cause confusion - I was just dealing with this recently when reviving benchmarks.

But, this is specific constant in specific test. Why would it need to be lifted somewhere?

Reply:
Ideally:
1. Make this a function with some path parameter
2. Make it check if the path exists and throw if doesn't (with a clear message)
3. In tests' parameters / attributes, leave the path unspecified, but give this Fanromas path as an example in the comments
4. Keep the tests commented out as they are now

I don't remember what are the latest xUnit capabilities on that, just sharing the vision :)

Reply:
Sorry gents, even more confused after this 😅.

This is how this plays out right now:
![image](https://github.com/dotnet/fsharp/assets/2621499/8822b376-2d9a-47dc-9fd5-ea038c4209bf)

Reply:
Well, AFAIU now if I just uncomment the test, it will start failing with a somewhat confusing message - I would need to dig down to understand this is happening because of the hardcoded paths.

But - not much of digging it will be, so you can leave it as it is.

Reply:
Oh, I see, well it is not that bad:

![image](https://github.com/dotnet/fsharp/assets/2621499/55eddea1-9550-4eda-a86f-f4653a2db3f3)

I feel like the comments around that test cover it for now.